### PR TITLE
feat(ui): layout component

### PR DESCRIPTION
**Pull request by 32penkin**
### Please read and mark the following check list before creating a pull request:

 - [] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/react-native-ui-kitten/blob/master/CONTRIBUTING.md) guide.
 - [] I read and followed the [New Feature Checklist](https://github.com/akveo/react-native-ui-kitten/blob/master/DEV_DOCS.md) guide.

 #### Short description of what this resolves:

### CODE REVIEW

File: src/framework/ui/index.ts [hunk 1]
```diff
@@ -7,14 +7,18 @@ import {
...
   Props as RadioGroupProps,
 } from './radioGroup/radioGroup.component';
+import {
+  Layout,
+  LayoutProps,
+} from './layout/layout.component';
 
 const Radio = styled<RadioComponent, RadioProps>(RadioComponent);
 const RadioGroup = styled<RadioGroupComponent, RadioGroupProps>(RadioGroupComponent);
+const StyledLayout = styled<Layout, LayoutProps>(Layout);
```
Comment (artyorsh) [inline src/framework/ui/index.ts]: Following convention in this index, it should be
```
import {
  Layout as LayoutComponent
}

const Layout = ...

export {
  ...
  Layout
}
```

File: src/framework/ui/index.ts [hunk 2]
```diff
@@ -7,14 +7,18 @@ import {
...
 
 export {
-  Radio,
-  RadioGroup,
-  RadioProps,
-  RadioGroupProps,
+  Radio, RadioProps,
+  RadioGroup, RadioGroupProps,
+  StyledLayout as Layout, LayoutProps,
```
Comment (artyorsh) [inline src/framework/ui/index.ts]: Each export on separate line, props in the end

File: src/framework/ui/layout/layout.component.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  View,
+  ViewProps,
+  StyleSheet,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+} from '@kitten/theme';
+
+interface LayoutComponentProps {
```
Comment (artyorsh) [inline src/framework/ui/layout/layout.component.tsx]: `LayoutProps`

File: src/framework/ui/layout/layout.component.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  appearance?: string | 'default';
+  children?: any;
+}
+
+export type LayoutProps = LayoutComponentProps & StyledComponentProps & ViewProps;
```
Comment (artyorsh) [inline src/framework/ui/layout/layout.component.tsx]: `Props`

File: src/playground/src/app.component.tsx [hunk 5]
```diff
@@ -30,7 +30,7 @@ export default class App extends React.Component<any, State> {
   }
 
   render() {
-    const { RadioScreen: RootScreen, ...nestedScreens } = Screens;
+    const { HomeScreen: RootScreen, ...nestedScreens } = Screens;
     const Router = withNavigation(RootScreen, nestedScreens);
 
     return (
```
Comment (artyorsh) [inline src/playground/src/app.component.tsx]: This file could be not tracked

File: src/playground/src/ui/screen/layout.component.tsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  render() {
+    return (
+      <View style={this.props.themedStyle.container}>
+        <LayoutComponent style={this.props.themedStyle.layout}>
+          <Text>Layout</Text>
+        </LayoutComponent>
+      </View>
+    )
+  }
+
```
Comment (artyorsh) [inline src/playground/src/ui/screen/layout.component.tsx]: blank line linter issue :)

**Comment by artyorsh** [comment]
Also my suggestion is to rename pr to `feat(ui): ...`

File: src/framework/ui/layout/layout.component.tsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  ViewProps,
+  StyleSheet,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+} from '@kitten/theme';
+
+interface LayoutComponentProps {
+  appearance?: string | 'default';
```
Comment (malashkevich) [inline src/framework/ui/layout/layout.component.tsx]: Remove hardcoded value

File: src/framework/ui/layout/layout.component.tsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  getComponentStyle = (style: StyleType): StyleType => ({
+    layout: style,
+  });
+
+  render() {
+    const componentStyle = this.getComponentStyle(this.props.themedStyle);
+
+    return (
+      <View {...this.props}>
+        <View style={[componentStyle.layout, styles.layout]}>
```
Comment (malashkevich) [inline src/framework/ui/layout/layout.component.tsx]: Make one `View` instead of two.

File: src/framework/ui/layout/layout.spec.config.ts [hunk 9]
```diff
@@ -0,0 +1,31 @@
+export const mapping = {
```
Comment (artyorsh) [inline src/framework/ui/layout/layout.spec.config.ts]: * types are missing across whole file
* linter is not applied 🤔

File: src/playground/src/ui/screen/layout.component.tsx [hunk 10]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```
Comment (artyorsh) [inline src/playground/src/ui/screen/layout.component.tsx]: linter is not applied 🤔

**Pull request merged**